sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -217,14 +158,13 @@ def from_blob_url(cls, blob_url, credential=None, snapshot=None, **kwargs):
             raise ValueError("Invalid URL. Provide a blob_url with a valid blob and container name.")
 
         path_snapshot, _ = parse_query(parsed_url.query)
-        if snapshot:
```

Comment:
>snapshot [](start = 11, length = 8)

if we delete this, then will None.snapshot throw AttributeError or other error type?